apps/server/src/modules/school/domain/type/ldap-config.ts

```diff
@@ -0,0 +1,41 @@
+import { EntityId } from '@shared/domain/types/entity-id';
+
+export interface LdapConfig {
+	active?: boolean;
```

Comment:
Wie sorgen wir dafür das wir Änderungen mit bekommen die im legacySchool noch passieren?
Oder ist das nicht wichtig, bis wir legacy in das school module integrieren?

Reply:
Hab die Systems betreffenden Sachen aus dem PR genommen, siehe anderen Kommentar.